### DISCUSSION TITLE
How to understand overview and score statistics

### DISCUSSION THREAD

**Discussion by dorinand** [iSoron/uhabits#1273] [General Discussion] [answered]
Hello everyone,

at first, I have to say big thanx for this app, it is simple to use, I really appreciate it :+1: . I am using this app for few weeks and I decide to evaluate my results but I find out, the overview and score is not very clear for me. I brought some examples.

**Gym Habbit**

I am doing some workout 4 times per week (at least, thats the goal). The score is **38%**. The graph shows that I am moving between **35-70%** but I am not sure, what this numbers show to me and how is it calculated. When I think about the last week, there is 3 activities registered in History section. The goal is 4 times/week so I would expect, the Score section would show me 3/4 = **75%** for the last week. In the Score board, it is something below **40%** (I assume it is the **38%** metioned in Overview section). How this numbers should be interpreted and how are they calculated?

**TestBook Habbit**

I create testing habbit for reading the book. I set it to 60pages/week. I manualy set that every Sunday, I red exactly 60 pages. There is no overview section but in Score section, I can see my score is about **60%**. I would expected **100%**, because my goal has been fulfilled. 

Can somebody explain me this behaviour? And why there is no Overview section?

![Habits-Gym](https://user-images.githubusercontent.com/27764640/149658682-784dc497-8af7-4ef3-995f-99715d0b8a8d.jpg)![Habits-TestBook](https://user-images.githubusercontent.com/27764640/149659240-fe10c354-273a-47cf-a5ae-7009595e0369.jpg)

**Comment by mattsoak** [comment, marked as answer]
Hello @dorinand !

The score is cumulative, not per day/week. Here's a link to the FAQ which includes details on how the score is calculated:
https://github.com/iSoron/uhabits/discussions/689

**Reply by dorinand** [reply]
Hi @mattsoak , thank you for your reply. I can see, there is exponential smoothing method used from the FAQ. But what that means for the result, how can I interpret it properly for myself?

Currently, I have **64%** score on my **Gym** habit. I have data from 17.9.2021. How can I interpret this, am I doing good job or a bad one? My testing habit **TestBook** has currently score **40%**. I ingest data only 8 times as you can see in the picture. For me, it is not clear, what that means for me, I am trying to do gym 4 times per week and my score is **64%**, I did not read a page for last 1.5 months (last data from 16.1.2022) and my score is **40%**. That is not a big diff between habit I am doing (almost) regularly and habit I stopped doing.
